.dockerignore

```diff
@@ -0,0 +1,10 @@
+.git/
+.dockerignore
+**/.gitignore
+**/c.out
+**/coverage.html
+**/Dockerfile
+**/README.md
+**/*_test.go
+**/Makefile
+awscodecommit/awscodecommit
```

Comment:
I don't understand why here only `awscodecommit/awscodecommit` built binary may be all other sources binaries should be under this `.dockerignore` too?

Reply:
good catch. it should list all the binaries. Will update the PR